parachain/pallets/commitments/src/tests.rs

```diff
@@ -64,3 +64,13 @@ fn test_add_message() {
 
 	});
 }
+
+
+#[test]
+fn test_make_offline_key() {
```

Comment:
offline_key or offchain_key?

Reply:
Actually I removed the test as I was only using it to print the key